client/src/utils/localization.js

```diff
@@ -1,47 +1,53 @@
-// eslint-disable-next-line no-undef
```

Comment:
This one wasn't just a comment to be removed; it was an eslint directive.  If you put this one back it'll lint cleanly again.